Modules/xASL_module_Population.m

```diff
@@ -286,18 +276,9 @@
                 x.S.InputNativeSpace = 1;
                 x.S.InputAtlasNativeName = [x.S.Atlases{iAtlas} '_Atlas'];
                 xASL_wrp_GetROIstatistics(x);
-                % ROI statistics (optional: native space)

```

Comment:
Where did the native space analysis with x.S.bSubjectSpecificROI==false go? This should be the default if you want native space. The addition for subject-specific lesions is good, but is a specific situation. E.g., the use case of a standard atlas with native space PVC is much more common.

Reply:
I don't understand. 
this change here isn't about bSubjectSpecificROI